code/crates/network/Cargo.toml

```diff
@@ -19,7 +19,7 @@ malachitebft-discovery = { workspace = true }
 malachitebft-metrics = { workspace = true }
 malachitebft-peer = { workspace = true }
 malachitebft-sync = { workspace = true }
-
+malachitebft-config = {workspace = true}
```

Comment:
Let's avoid introducing a dependency on `malachite-config` for now, and define a copy of the `ProtocolNames` struct in the `network` crate, like we do for other settings.

We will probably revisit this at some point, but let's keep it consistent with the existing behavior for now.